### PR TITLE
Add PHP 8.3

### PR DESCRIPTION
**Pull request by SuperDJ**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "extra": {
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
```
Comment (Ocramius) [inline composer.json:30]: `composer.lock` needs updating (try with `composer update nothing`)

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by SuperDJ** [comment]
This PR requires https://github.com/laminas/laminas-xml/pull/17

**Comment by Ocramius** [comment]
@SuperDJ if you give this a rebase on top of `2.19.x`, we should be green, and this could probably be merged + released :)

**Comment by Xerkus** [comment]
Superseded by #39

**Pull request closed without merging**